index.js

```diff
@@ -0,0 +1,42 @@
+const langProp = require('./src/languageProperties.json');
+const { handleRequest } = require('./src/rextester.js');
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const { port } = require('./config.json');
+
+
+app.use(bodyParser.json() );
+app.use(bodyParser.urlencoded( {
+    extended: true,
+} ) );
+
+app.get('/languages', (req, res) => {
+    const langArray = [];
+    let currentLangAliases,
+        currentLangImports;
+    langProp.languageProperties.forEach( (i) => {
+        currentLangAliases = i.aliases.join(', ');
+        currentLangImports = i.defaultImports.join(', ');
+        langArray.push(`Language: ${i.commonName};\nAliases: ${currentLangAliases};\nImports: ${currentLangImports}`);
+    } );
+    return res.send(langArray.join('\n\n') ).end();
+} );
+
+app.post('/', async(req, res) => {
```

Comment:
This doesn't need to be async for now.
You don't need to await handleRequest either